dvc/command/plots.py

```diff
@@ -148,15 +145,19 @@ def add_parser(subparsers, parent_parser):
         "--targets",
         nargs="*",
         help="Plots file to visualize. Shows all plots by default.",
+        metavar="<path>",
     )
     plots_diff_parser.add_argument(
-        "revisions", nargs="*", default=None, help="Git commits to plot from",
+        "revisions",
+        nargs="*",
+        default=None,
+        help="Git commits to plot from/to",
```

Comment:
We do not write plots to commits. I might miss something, why this `to`?

Reply:
That was my fault. Our latest version is

> Git commits to find metrics to compare

per https://github.com/iterative/dvc.org/pull/1382#pullrequestreview-427498405